tools/bowtie2/bowtie2_wrapper.xml

```diff
@@ -442,7 +449,29 @@
             </actions>
         </data>
         
-        <data format="bam" name="output" label="${tool.name} on ${on_string}: aligned reads (sorted BAM)">
+        <data format="bam" name="output_1" label="${tool.name} on ${on_string}: aligned reads (sorted BAM)">
```

Comment:
Probably it would be better to still call this `output` for backward compatibility with existing workflows.